Hydration_freenrg/bash_scripts/analyse_all.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#SBATCH -n 1
+    # Request 1 gpu per job
+#SBATCH --gres=gpu:1
+    # allocate 5 processors/CPUs per GPU
+#SBATCH --cpus-per-gpu 5
+    # allocate 5 GB of memory per job
+#SBATCH --mem 5000
+    # Set job name
+#SBATCH --job-name=analysis
+#SBATCH --output=./scriptouts/analysis.o
+#SBATCH --error=./scriptouts/analysis.err
+
+source /home/matthew/mambaforge/bin/activate openbiosim
+python3 --version
+cd ../
+maindir="$PWD"
+echo "master dir is $maindir"
+for filename in ./Systems/*; do
+        cd $filename
+        echo "System directory = $PWD"
+	systemfolder="$PWD"
+	for rep in ./rep*; do
+		cd $rep/solv
+		echo "$PWD"
+		cd lambda_0.0000
+		python ~/mambaforge/envs/openbiosim/share/Sire/scripts/lj-tailcorrection.py -t somd.prm7 -c somd.rst7 -m somd.pert -C somd.cfg -l 0.00 -r traj000000001.dcd -s 1 1> ../freenrg-LJCOR-lam-0.000.dat 2> /dev/null
```

Comment:
lj-tailcorrection should be in the path if the environment has been activated. the code assumes that the environment openbiosim has been created in mamba which may not be portable.

Reply:
Changed this file to use the binaries rather than the script, should now be a bit more portable.